src/Comps/graphCard/style.ts

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import { COLORS } from "../../globalStyle";
+
+export const CardContainer = styled.div`
+  position:relative;
+
+  width: 412px;
+  height: 355px;
+  background: #ffffff;
+  border: 1px solid #d9dbe9;
+  box-shadow: 0px 32px 64px rgba(0, 0, 0, 0.05);
+  border-radius: 20px;
+`;
+
+export const Title = styled.p`
+  position: absolute;
```

Comment:
create a typography file to your project containing all the font size and attributes.
then for example,  you can just use it like <H2Bold> some bold text <H2Bold/>.

Always ask your self, what would happen if tomorrow you get a task of changing all the font family in the system for example. There should be only 1 place where the global colors, font size, font family, font weight etc are configured